logic/performance-comparison/userCodeExecutor.js

```diff
@@ -0,0 +1,46 @@
+import executeVmCode from "@utils/vmCodeExecutor";
+
+function validateCode(userResult) {
```

Comment:
```suggestion
function validateExcutedResultType(userResult) {
```
유저가 입력한 코드를 실행한 반환값을 확인하여 `object` 타입일 경우 에러메세지를 반환하는 함수라고 이해했는데요!

제가 이해한 부분이 맞다면 `validateCode`라는 함수명은 "**코드를 검증하다**"라는 의미로 해석되어 
`validateExcutedResultType` 처럼 **조금 더 명확한 함수명**으로 수정하는 것이 좋다고 생각됩니다!

Reply:
함수 이름이 명확하지 않아 의미 전달이 잘못된것같습니다.

`object` 타입일 경우 key값중 `error`, `errorStackMessage`가 둘 다 있다면 `true`, 아니라면 `false`를 반환하는 함수인데
`validateErrorResult`라는 이름 어떨까요?

Reply:
> object 타입일 경우 key값중 error, errorStackMessage가 둘 다 있다면 true, 아니라면 false를 반환하는 함수

저도 함수를 정확하게 이해하지 못 했었네요 🙇🏻
`validateErrorResult` 함수명 좋은 것 같습니다!